1.8/administration/logging/elk.md

```diff
@@ -1,147 +1,200 @@
 ---
-post_title: Log Management with ELK
-nav_title: ELK
+post_title: Log Management in DC/OS with ELK
 menu_order: 1
+enterprise: 'yes'
 ---
-You can pipe system and application logs from a DC/OS cluster to your existing ElasticSearch, Logstash, and Kibana (ELK) server.
+
+You can pipe system and application logs from the nodes in a DC/OS cluster to your existing ElasticSearch, Logstash, and Kibana (ELK) server. This article recommends to store all unfiltered logs directly on ElasticSearch, and perform filtering/specialized querying on ElasticSearch directly. Consequently, the suggested pipeline sends the output from filebeat in the nodes directly to a centralized ElasticSearch instance, without using Logstash. If you're interested in using Logstash for log processing or parsing, consult the [Filebeat][2] and [Logstash][8] documentation.
```

Comment:
Suggest: `The filebeat output from each node is sent directly to a centralized ElasticSearch instance, without using Logstash.`